### PR TITLE
SC - 6816 - new test @editClass

### PR DESCRIPTION
**Pull request by Grzesiek80**
# Description
crete new test editClass

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/SC-6816

### CODE REVIEW

File: features/management_classes/editClass.feature [hunk 1]
```diff
@@ -15,4 +15,27 @@ Feature: I want to edit a class
...
+		When <userRole> logs in with email '<username>' and password '<password>'
+		And <userRole> goes to management
+		And <userRole> goes to class management
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher1Lastname>' is visible
+		When <userRole> chooses class with name '<className>' and click managemnt class
+		And <userRole> adds a teacher to class with name '<teacher>'
+		And <userRole> adds a student to class with name '<student>'
+		And <userRole> clicks on Save-changes in class button
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher1Lastname>' is visible
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher2Lastname>' is visible
```
Comment (yuliashikhareva) [inline features/management_classes/editClass.feature:31]: duplicated step

File: features/management_classes/editClass.feature [hunk 2]
```diff
@@ -15,4 +15,27 @@ Feature: I want to edit a class
...
\ No newline at end of file
+			| teacher  | karl.teacher.qa@schul-cloud.org | Schulcloud1qa!  | 8a              | 4d                 | 2021/22    | Herzog          |
+
+	@editClassAddStudentTeacherAndDeleteStudentTeacher
+	Scenario Outline: As a user, I want to be able to edit a class
+		When <userRole> logs in with email '<username>' and password '<password>'
+		And <userRole> goes to management
+		And <userRole> goes to class management
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher1Lastname>' is visible
+		When <userRole> chooses class with name '<className>' and click managemnt class
```
Comment (yuliashikhareva) [inline features/management_classes/editClass.feature:26]: clicks Class-management (or 'Class-management'?)

File: features/management_classes/editClass.feature [hunk 3]
```diff
@@ -15,4 +15,27 @@ Feature: I want to edit a class
...
+		And <userRole> adds a teacher to class with name '<teacher>'
+		And <userRole> adds a student to class with name '<student>'
+		And <userRole> clicks on Save-changes in class button
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher1Lastname>' is visible
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher2Lastname>' is visible
+		Then <userRole> should see that class with name '<className>' and '<numberOfStudents>' students is visible
+		When <userRole> chooses class with name '<className>' and click managemnt class
+		And <userRole> delete teacher '<teacher>' from class
+		And <userRole> delete student '<student>' from class
```
Comment (yuliashikhareva) [inline features/management_classes/editClass.feature:35]: deletes

File: page-objects/pages/managementPages/ManageClassesPage.js [hunk 4]
```diff
@@ -19,8 +25,45 @@ async function clickEditClassBtn() {
...
+        let classItem = await classNamePromise.getText();
+		if (classItem === className) {
+			let numberOfStudents = await elementHelpers.getElementText('tr:nth-child('+ index +') > td:nth-child(4)');
+            expect(numberOfMembers).to.equal(numberOfStudents)
+			break;
+		}
+    }
+}
+    // choose between className, teachers, students, schoolYear
+ async function getClassDetailsList(whichCell) {
```
Comment (yuliashikhareva) [inline page-objects/pages/managementPages/ManageClassesPage.js:56]: probably find another name for parameter?

File: features/management_classes/editClass.feature (hunk 1 above)
Comment (Grzesiek80) [reply features/management_classes/editClass.feature:31]: the same step but different parameter is used (teacher)

File: features/management_classes/editClass.feature (hunk 2 above)
Comment (Grzesiek80) [reply features/management_classes/editClass.feature:26]: done

File: features/management_classes/editClass.feature (hunk 3 above)
Comment (Grzesiek80) [reply features/management_classes/editClass.feature:35]: done

File: page-objects/pages/managementPages/ManageClassesPage.js (hunk 4 above)
Comment (Grzesiek80) [reply page-objects/pages/managementPages/ManageClassesPage.js:56]: done

**Review by yuliashikhareva — APPROVED** [review]

**Review by d3n153 — APPROVED** [review]
Fine from my side.

**Pull request merged**